index.html

```diff
@@ -9,5 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+
+    <div class="title-container">
+        <img width="50px" height="50px" src="./assets/triangle.svg" alt="triangle">
```

Comment:
If you look at the card image 
<img width="516" alt="captura de pantalla 2018-05-30 a la s 12 20 26" src="https://user-images.githubusercontent.com/36163179/40733768-7d43836e-6404-11e8-8bec-29cc58c00f37.png">

And then you see your development then you will find a difference
<img width="213" alt="captura de pantalla 2018-05-30 a la s 12 21 15" src="https://user-images.githubusercontent.com/36163179/40733580-fcd130e6-6403-11e8-83d4-c9f4039635c9.png">

`The main difference is that the technology title like 'Front-End Tools' in the card is 'above' the triangle img and in your development is next to it.`

The main idea of this card is to use pseudoelements... ` investigate and use the '&::before' ` in order to handle that behaviour